CHANGELOG.md

```diff
@@ -35,6 +33,24 @@ Breaking changes
 - x/nft is using schema versioned model and messages
 - x/validators is using schema versioned model and messages
 
+## 0.14.0
+- Tests were cleaned up and no use testify or convey packages. A new package
+  `weavetest/assert` contains test helpers
+- Simplify transaction message unpacking with weave.LoadMsg
+- Initial version of governance model
+- crypto package improvements #495
+- Weave documentation : References to IOV-CORE examples need removing #464
+- Reduce timeout resolution in client/backend interface #447
+- Move timeout point to expired state (for atomic swaps) #446
+- Simplify validation logic in Handlers #437
+- Charge gas in x/sigs #429
+- New bump sequence Tx #388
+- Anti-spam fee changes    
+  
+Breaking changes  
+  
+- Move Memo and ChainID to top level Tx codec #435
```

Comment:
This never happened, right?

Reply:
good catch, just got there from our auto-generated changelog..